fury/material.py

```diff
@@ -29,39 +29,63 @@ def manifest_pbr(actor, metallicity=1, roughness=.5):
     prop.SetRoughness(roughness)
 
 
-def manifest_standard(actor, ambient_level=.7, diffuse_level=.8,
-                      specular_level=.5, specular_power=10,
-                      interpolation='phong'):
+def manifest_standard(actor, ambient_level=0, ambient_color=(1, 1, 1),
+                      diffuse_level=1, diffuse_color=(1, 1, 1),
+                      specular_level=0, specular_color=(1, 1, 1),
+                      specular_power=1, interpolation='gouraud'):
     """Apply the standard material to the selected actor.
 
     Parameters
     ----------
     actor : actor
     ambient_level : float, optional
-        Metallic or non-metallic (dielectric) shading computation value. Values
-        must be between 0.0 and 1.0.
+        Ambient lighting coefficient. Value must be between 0.0 and 1.0.
+    ambient_color : tuple (3,), optional
+        Ambient RGB color where R, G and B should be in the range [0, 1].
     diffuse_level : float, optional
-        Parameter used to specify how glossy the actor should be. Values must
-        be between 0.0 and 1.0.
+        Diffuse lighting coefficient. Value must be between 0.0 and 1.0.
+    diffuse_color : tuple (3,), optional
+        Diffuse RGB color where R, G and B should be in the range [0, 1].
     specular_level : float, optional
-        Parameter used to specify how glossy the actor should be. Values must
-        be between 0.0 and 1.0.
+        Specular lighting coefficient. Value must be between 0.0 and 1.0.
+    specular_color : tuple (3,), optional
+        Specular RGB color where R, G and B should be in the range [0, 1].
     specular_power : float, optional
-        Parameter used to specify how glossy the actor should be. Values must
-        be between 0.0 and 1.0.
-    interpolation : float, optional
-        Parameter used to specify how glossy the actor should be. Values must
-        be between 0.0 and 1.0.
+        Parameter used to specify the intensity of the specular reflection.
+        Value must be between 0.0 and 128.0.
+    interpolation : string, optional
+        If 'flat', the actor will be shaded using flat interpolation. If
+        'gouraud' (default), then the shading will be calculated at the
```

Comment:
Make sure you accept Gouraud and Phong (or a fully capitalized version of those).